angr/procedures/libc/snprintf.py

```diff
@@ -15,7 +15,10 @@ def run(self, dst_ptr, size):  # pylint:disable=arguments-differ,unused-argument
         # The format str is at index 2
         fmt_str = self._parse(2)
         out_str = fmt_str.replace(3, self.arg)
-        self.state.memory.store(dst_ptr, out_str)
+        try:
+            self.state.memory.store(dst_ptr, out_str)
+        except:
```

Comment:
?

Reply:
Probably a leftover in Nilo's original branch. Should be removed.